.editorconfig

```diff
@@ -0,0 +1,15 @@
+# This file is for unifying the coding style for different editors and IDEs
+# editorconfig.org
+
+root = true
+
+[*]
+indent_style = tab
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
+
+[*.yml]
+indent_style = space
+indent_size = 2
```

Comment:
FWIW, the package.json is currently using spaces.

Reply:
@timmywil It does not, I reformatted it in this PR

Reply:
Ah, there was a `test/package.json` file with 3 lines that used spaces for its single indent. Fixed now. But the main `package.json` I already updated before submitting the PR.

Reply:
I think I had whitespace turned off in the diff